src/exit.rs

```diff
@@ -64,6 +64,10 @@ pub enum AxVCpuExitReason {
         addr: GuestPhysAddr,
         /// The width of the MMIO read.
         width: AccessWidth,
+        /// dst read reg idx
+        reg: usize,
+        /// dst red reg width
+        reg_width: AccessWidth,
     },
     /// The instruction executed by the vcpu performs a MMIO write operation.
     MmioWrite {
```

Comment:
One more thing, please modify the type of EnternelInterrupt's argument from u8 to u64